npm/vite-plugin-cypress-esm/cypress.config.ts

```diff
@@ -17,7 +17,7 @@ export default defineConfig({
               jsxRuntime: 'classic',
             }),
             CypressEsm({
-              ignoreList: ['*Immutable*', '*MyAsync*'],
+              ignoreList: ['**/*ignoreList.cy.ts', '*MyAsync*'],
```

Comment:
Do we need to add an example with React Hot Refresh or it's not necessary? Do we have a test for this new line: https://github.com/cypress-io/cypress/pull/26714/files#diff-a7f0976b3c78a92186ca9729b261b946ddd9cd67197470ca0d52f12fbfbb3a34R76

Reply:
Whoops, we are missing a test for that. Will get one added

Reply:
Added one each for static vs dynamic syntax

Reply:
thanks!